packages/utilities/src/object.ts

```diff
@@ -93,6 +93,7 @@ export function mapEnumByName<T>(
         // if the property is not just a number (because enums in TypeScript will map both ways)
         return callback(p as string, theEnum[p]);
       }
+      return undefined;
```

Comment:
is this needed? returning undefined is the same as not returning anything.

Reply:
Returning nothing has `void` return type. Returning nothing will generate a TS error if `void` is not declared as the function's return type. It forces you to explicitly return something.

Reply:
Similar thing here, v0's parsing was complaining about not all paths returning something, adding `return undefined` here allows us to mitigate this warning while maintaining the logic.